lib/prj/prj_make_META.ml

```diff
@@ -0,0 +1,71 @@
+open Core.Std
+open Core_extended.Std
+open Async.Std
+
+let format_cmds cmds =  "[\n" ^
+                        (List.fold ~init:""
+                           ~f:(fun acc el -> acc ^ "    [" ^ el ^ "]\n") cmds) ^
+                        "]\n"
+
+let format_build_cmds = function
```

Comment:
Couldn't these three functions be just one? The only thing that changes is the word after `"make ..."`